### DISCUSSION TITLE
Analysis shows no result for Repo

### DISCUSSION THREAD

**Discussion by odin568** [github-linguist/linguist#5286] [Classification] [answered]
Hi all,
I have quite some similar repositories, mainly Java applications.
On all but one, the classification works fine. Just on one I don't get any result:
https://github.com/FFW-Baudenbach/FE2_Zerlegung_ILSAnsbach
First I thought it was because of the checked-in jar  file in /libs folder. Renamed it to /distributions so that it should get ignored. I don't see now any other error. Only thing that all java files are in folder/namespace /../external/../ but I don't see a regex for linguist which has a problem there.
Can somebody help?
Also tried to add a .gitattributes to classify everything in /src to java but that did also not help.
Thanks and best regards!

**Comment by lildude** [comment, marked as answer]
> Only thing that all java files are in folder/namespace /../external/../ but I don't see a regex for linguist which has a problem there.

This is precisely why:

https://github.com/github/linguist/blob/2a6f9e29e198afbaa721164680bbc3b292dad260/lib/linguist/vendor.yml#L111

You'll need to mark that path or those files an "unvendored" for them to be considered.

**Reply by odin568** [reply]
ah ok. My feeling said that already but I did not find it in the regex. Seems I did not look close enough ;) Thanks a lot for your hint :-) I'll apply the unvendored solution

**Reply by odin568** [reply]
`**/external/** linguist-vendored=false`
resolved it!
